### DISCUSSION TITLE
Discord Ban Appeal

### DISCUSSION THREAD

**Discussion by Saturna19-fr** [pterodactyl/panel#5158] [General Community]
Hello

I was banned from your Discord, but I don't know why.

Can you explain it to me?

Best regards
@saturna19

**Comment by danny6167** [comment]
The bot told you why you were banned when the ban was placed.
You were banned because you keep coming back trying to get support for someone else's panel after the moderators told you we can't help you with it. You need to talk to the panel owner.
